src/sonic-eventd/tools/events_monit_test.py

```diff
@@ -60,28 +60,32 @@ def map_dict_fvm(s, d):
 def test_receiver(event_obj, cnt):
     global rc_test_receive
 
-    sh = events_init_subscriber()
+    sh = events_init_subscriber(false, 6000, null)
```

Comment:
We don't want 6 seconds
1 second will do
declare this as constant

RECEIVE_TIMEOUT = 1000
and use that here.